_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -50319,6 +50654,20 @@
 /obj/machinery/door/firedoor,
 /turf/open/floor/plasteel/dark,
 /area/library)
+"vYv" = (
+/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden{
+	dir = 8
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 8
+	},
+/obj/machinery/light_switch{
+	dir = 8;
+	pixel_x = -20;
+	pixel_y = 0
```

Comment:
dv

Reply:
How do I fix them?

Reply:
@ShizCalev How do I fix dirty vars?

Reply:
You have to edit them out in Notepad++ or another editor.

Reply:
So just remove the "pixel_y = 0" etc?

Reply:
Yes, and make sure the previous line loses its semicolon.